### ISSUE TITLE
Grover::Error - SyntaxError: Unexpected token '?'`

### ISSUE THREAD

**Issue by karekauan** [Studiosity/grover#250]
I'm migrating the pdfs from the Prawn system to Grover, but when trying to run the basic example:

```
 grover = Grover.new('https://google.com', format: 'A4')
 pdf = grover.to_pdf
 send_data(
     pdf,
     filename: report_name,
     type: 'application/pdf',
     disposition: 'attachment'
 )
```

 I received the following error on the line 2:

```Grover::Error - SyntaxError: Unexpected token '?'```

The versions used are: 
* Ruby 3.2.0
* Rails 7.0.8.3
* Grover 1.1.9
* Puppeteer 22.15.0

Is there any incompatibility with any of these versions?

**Comment by abrom** [comment]
No, but you'll want to check the NODE version you're using.. I think you'll find it's one that is very very out of date (that does not support safe navigators ie < v14).

Note, Puppeteer 22.15.0 wants node v20.8.4+ 😉

**Comment by JennsiS** [comment]
Had the same issue with:

- Ruby 3.2.3
- Rails 7.1.2
- Puppeteer: 21.7.0
- Node 19.0.0
- Grover 1.1.6

It was working before, and now stop working

**Comment by abrom** [comment]
The above error is specific to Node versions that don't support the safe navigator operator (ie < v14). It may be that you have multiple node versions installed on your system?

You can validate this by trying to call to node from within your ruby environment. eg:
```
%x{node -v}
```
